app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>BlogApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
```

Comment:
For this milestone, you should create associations between models. I should see `User`, `Post`, `Comment`, and `Like` models inside the `app/models/` directory. You currently have nothing there.

For example:

- Under `app/models/user.rb` you should have something similar to this:
```
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```
- Under `app/models/post.rb` you should have something similar to this:
```
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
```

- Kindly add relationships and associations for `comments` under `app/models/comment.rb` according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

- Kindly add relationships and associations for `likes` under `app/models/like.rb` according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).